packages/design-system/__mocks__/events.ts

```
@@ -0,0 +1,22 @@
+export class EventEmitter {
+  private events: Record<string, Function[]> = {}
```

Biome (1.9.4)

[error] Don't use 'Function' as a type.

Prefer explicitly define the function shape. This type accepts any function-like value, which can be a common source of bugs.

(lint/complexity/noBannedTypes)